app/javascript/gobierto_dashboards/modules/contracts_controller.js

```diff
@@ -19,7 +19,8 @@ Vue.use(VueRouter);
 Vue.config.productionTip = false;
 
 // Global variables
-let data, reduced, ndx, _amountRange, vueApp, charts = {};
+let data, reduced, ndx, _amountRange, vueApp, unfilteredTendersData, charts = {};
+const tendersFilters = {submission_date: [], process_type: [], contract_type: [] };
```

Comment:
Tell me this is not the branch you went to move this global variables into the `this` object, because the controller it's a javascript class

Reply:
No, it's not! this is already moved to class variables in the most recents class 👍